.github/workflows/contrib.yml

```diff
@@ -0,0 +1,13 @@
+name: Core Contrib Test
+
+on:
+  push:
+    branches-ignore:
+    - 'release/*'
+  pull_request:
+
+jobs:
+  contrib_0:
+    uses: emdneto/opentelemetry-python-contrib/.github/workflows/core_contrib_test_0.yml@generate-workflows-core
```

Comment:
Note: change this after reviews and approvals. Please do not resolve this comment

Reply:
The branch name should be changed anyway for release branches right?

Reply:
Right. This is a trial to avoid the manual sync every time tox.ini changes in the contrib repo.

Reply:
So merge https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3035, then merge this, then trial? Then, if successful, change branch and readmes too?

Reply:
@tammy-baylis-swi Merge contrib#3035 and after that I'll change this PR to point to open-telemetry/contrib repo instead of my fork

Reply:
Thanks 😃